uint32 time
float32 omega_x
float32 omega_y
float32 omega_z
float32 accel_x
float32 accel_y
float32 accel_z

